### DISCUSSION TITLE
How works achivements

### DISCUSSION THREAD

**Discussion by firefirefly3141** [community/community#19010] [Other Feature Feedback, Questions, & Ideas] [answered]
Someone can explain how works achivements?

**Comment by rflmedina** [comment, marked as answer]
Official explanation:[ Github Achivements](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/)

**Reply by queenofcorgis** [reply]
Thank you for sharing that @rafaelaxmedina ❤️
